pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -1025,7 +968,7 @@ func TestReconcile(t *testing.T) {
 				Obj(),
 			checks:      []kueue.AdmissionCheck{*baseCheck.DeepCopy()},
 			flavors:     []kueue.ResourceFlavor{*baseFlavor1.DeepCopy(), *baseFlavor2.DeepCopy()},
-			configs:     []kueue.ProvisioningRequestConfig{*provReqConfigWithBaseBackoff(provReqConfigWithRetryLimit(baseConfig, 1), 0)},
+			configs:     []kueue.ProvisioningRequestConfig{*baseConfigWithRetryStrategy.Clone().RetryLimit(1).BaseBackoff(0).DeepCopy()},
```

Comment:
```suggestion
			configs:     []kueue.ProvisioningRequestConfig{baseConfigWithRetryStrategy.Clone().RetryLimit(1).BaseBackoff(0).Obj()},
```
Please follow this patter with the rest of changes